pkg/ui/src/views/cluster/containers/nodeGraphs/dashboards/runtime.tsx

```diff
@@ -99,5 +99,21 @@ export default function (props: GraphDashboardProps) {
         <Metric name="cr.node.sys.cpu.sys.ns" title="Sys CPU Time" nonNegativeRate />
       </Axis>
     </LineGraph>,
+
+    <LineGraph title="Clock Offset" sources={nodeSources}
+    tooltip={`Mean clock offset of each node against the rest of the cluster.`}>
+      <Axis label="offset" units={AxisUnits.Duration}>
+        {
+          _.map(nodeIDs, (nid) => (
+            <Metric
+              key={nid}
+              name="cr.node.clock-offset.meannanos"
+              title={nodeDisplayName(nodesSummary, nid)}
+              sources={storeIDsForNode(nodesSummary, nid)}
+            />
+          ))
+        }
+      </Axis>
+    </LineGraph>,
```

Comment:
nit: plz reindent as

```js
<LineGraph
  title="Clock Offset"
  sources={nodeSources}
  tooltip={`Mean clock offset of each node against the rest of the cluster.`}
>
  <Axis label="offset" units={AxisUnits.Duration}>
    {
      _.map(nodeIDs, (nid) => (
        <Metric
          key={nid}
          name="cr.node.clock-offset.meannanos"
          title={nodeDisplayName(nodesSummary, nid)}
          sources={storeIDsForNode(nodesSummary, nid)}
        />
      ))
    }
  </Axis>
</LineGraph>,
```

Maybe soon we will use [prettier](https://github.com/prettier/prettier) to catch/fix stuff like this…

Reply:
Done.

Reply:
oh jeez, hate to put you through the CI cycle again but it's still not right — lines 109-118 are outdented all the way, and should be indented under `<Axis>`. Lines 103-107 are good though 👍

Reply:
The pain of lack of lints!

Reply:
Weird, vim is lying to me:

<img width="601" alt="screen shot 2018-03-08 at 5 48 04 pm" src="https://user-images.githubusercontent.com/7085343/37180945-ec5b7d8a-22f8-11e8-9206-30390881b12b.png">

Reply:
Actually done this time.

Reply:
@jordanlewis it burns. Need to fix this…

@a-robinson wow that is weird!

Reply:
Looks good now.